utils/modularize.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * @author mrdoob / http://mrdoob.com/
+ */
+
+var fs = require( 'fs' );
+
+var srcFolder = '../examples/js/';
+var dstFolder = '../examples/jsm/';
+
+var files = [
+	{ path: 'controls/OrbitControls.js', ignoreList: [] },
+	{ path: 'controls/MapControls.js', ignoreList: [] },
+	{ path: 'controls/TrackballControls.js', ignoreList: [] },
+	// { file: 'controls/TransformControls.js', ignoreList: [] },
+	{ path: 'loaders/GLTFLoader.js', ignoreList: [ 'NoSide', 'Matrix2', 'DDSLoader', 'DRACOLoader', 'BufferGeometryUtils' ] },
+	{ path: 'loaders/OBJLoader.js', ignoreList: [] }
+];
+
+for ( var i = 0; i < files.length; i ++ ) {
+
+	var file = files[ i ];
+	convert( file.path, file.ignoreList );
+
+}
+
+//
+
+function convert( path, ignoreList ) {
+
+	var contents = fs.readFileSync( srcFolder + path, 'utf8' );
+
+	var className = '';
+	var dependencies = {};
+
+	// class name
+
+	contents = contents.replace( /THREE\.([a-zA-Z0-9]+) = /g, function ( match, p1 ) {
+
+		className = p1;
+
+		console.log( className );
+
+		return `_IMPORTS_\n\nvar ${p1} = `;
+
+	} );
+
+	contents = contents.replace( /THREE\.([a-zA-Z0-9]+)\./g, function ( match, p1 ) {
+
+		if ( p1 === className ) return `${p1}.`;
+
+		return match;
+
+	} );
+
+	// methods
+
+	contents = contents.replace( /new THREE\.([a-zA-Z0-9]+)\(/g, function ( match, p1 ) {
+
+		if ( ignoreList.includes( p1 ) ) return match;
+
+		dependencies[ p1 ] = true;
+
+		return `new ${p1}(`;
+
+	} );
+
+	// constants
+
+	contents = contents.replace( /THREE\.([a-zA-Z0-9]+)/g, function ( match, p1 ) {
+
+		if ( ignoreList.includes( p1 ) ) return match;
+		if ( p1 === className ) return match;
+
+		dependencies[ p1 ] = true;
+
+		// console.log( match, p1 );
+
+		return `${p1}`;
+
+	} );
+
+	//
+
+	var keys = Object.keys( dependencies ).sort().map( value => '\n\t' + value ).toString();
+	var imports = `import {${keys}\n} from "../../../build/three.module.js";`;
```

Comment:
I think it should import from individual module files in `src`, not from the build. Multiple import statements.

Reply:
That seems more likely to break the test proposed in https://github.com/mrdoob/three.js/pull/15518#issuecomment-451256036, unless the addon is part of the `three.module.js` build.

Reply:
Okay, I get it now. It imports from the exports of 'three'. That's fine then.

Reply:
Ideally it would import from source, but wanted to get something working.